helm/resource_release.go

```
@@ -617,6 +673,27 @@ func resourceDiff(d *schema.ResourceDiff, meta interface{}) error {
 		return nil
 	}
 
+	n := d.Get("namespace").(string)
+
+	helmConf, err := m.GetHelmConfiguration(n)
+	if err != nil {
+		return err
+	}
+
+	name := d.Get("name").(string)
+	r, err := getRelease(helmConf, name)
```

the err is not handled?